collector/receiver/otelarrowreceiver/README.md

```diff
@@ -1,89 +1,108 @@
-gRPC. The HTTP/JSON address is the same as gRPC as the protocol is recognized
-and processed accordingly. Note the serialization format needs to be [protobuf JSON](https://developers.google.com/protocol-buffers/docs/proto3#json).
+## Advanced Configuration
 
-The HTTP/JSON configuration also provides `traces_url_path`, `metrics_url_path`, and `logs_url_path`
-configuration to allow the URL paths that signal data needs to be sent to be modified per signal type.  These default to
-`/v1/traces`, `/v1/metrics`, and `/v1/logs` respectively.
+The endpoint is configurable:
 
-To write traces with HTTP/JSON, `POST` to `[address]/[traces_url_path]` for traces,
-to `[address]/[metrics_url_path]` for metrics, to `[address]/[logs_url_path]` for logs.
-The default port is `4318`.  When using the `otlphttpexporter` peer to communicate with this component,
-use the `traces_endpoint`,  `metrics_endpoint`, and `logs_endpoint` settings in the `otlphttpexporter` to set the
-proper URL to match the address and URL signal path on the `otlpreceiver`.
+- `endpoint` (default = 0.0.0.0:4317 for grpc protocol, 0.0.0.0:4318 http protocol):
+  host:port to which the receiver is going to receive data. The valid syntax is
+  described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
 
-### CORS (Cross-origin resource sharing)
+Several helper files are leveraged to provide additional capabilities automatically:
 
-The HTTP/JSON endpoint can also optionally configure [CORS][cors] under `cors:`.
-Specify what origins (or wildcard patterns) to allow requests from as
-`allowed_origins`. To allow additional request headers outside of the [default
-safelist][cors-headers], set `allowed_headers`. Browsers can be instructed to
-[cache][cors-max-age] responses to preflight requests by setting `max_age`.
+- [gRPC settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md) including CORS
+- [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 
-[cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-[cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
-[cors-max-age]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
+Keepalive settings are vital to the operation of OTel-Arrow, because
+longer-lived streams use more memory.  Since every stream of data is
+different, we recommend experimenting to find a good balance between
+memory usage and stream lifetime.
 
-```yaml
+```
 receivers:
-  otlp:
+  otelarrow:
     protocols:
-      http:
-        endpoint: "localhost:4318"
-        cors:
-          allowed_origins:
-            - http://test.com
-            # Origins can have wildcards with *, use * by itself to match any origin.
-            - https://*.example.com
-          allowed_headers:
-            - Example-Header
-          max_age: 7200
+      grpc:
```

Comment:
Nit - since you mention `endpoint` above, maybe it would be helpful to include it in this configuration block as an example

Reply:
PTAL. I've removed the requirement to list `grpc` at all and have revised this text.